posts/2021-04-30-dev-mode-with-aot.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Open Liberty development mode startup improved with AOT"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dsouzai
+author_github: https://github.com/dsouzai
+seo-title: Open Liberty development mode startup improved with AOT - OpenLiberty.io
+seo-description: Improved Open liberty deveopment mode startup with AOT compilation using Eclipse OpenJ9.
+blog_description: "Improved Open liberty deveopment mode startup with AOT compilation using Eclipse OpenJ9."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Open Liberty development mode startup improved with AOT
+Irwin D'Souza <https://github.com/dsouzai>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// // // // // // // //
```

Comment:
I think we can remove all the template comment lines

Reply:
Made requested changes (see https://github.com/OpenLiberty/blogs/compare/ed1c305e8fee72472f81e224a1a82d205afd5bab..83e09da74325b699dcaa22fe68b54e5f35ead76f)